tests/Handler/AddPrefixCommandHandlerTest.php

```diff
@@ -82,12 +82,61 @@ protected function tearDown()
 
     public function testAddPrefixToDirectory()
     {
-        chdir(dirname($this->tempDir));
+        chdir($this->tempDir);
```

Comment:
This is the reason why the tests fail on AppVeyor. On Windows, you cannot remove directories that are or contain the current working directory you should:

* store the current working directory in `setUp()` (with `getcwd()`)
* restore the working directory in `tearDown()` before you delete the temporary directory